### PR TITLE
Extend getpeerinfo result with current block height.

### PR DESCRIPTION
**Pull request by Roasbeef**
Related to: btcsuite/btcd#296

### CODE REVIEW

File: jsonresults.go [hunk 1]
```diff
@@ -151,6 +151,7 @@ type GetPeerInfoResult struct {
 	SubVer         string  `json:"subver"`
 	Inbound        bool    `json:"inbound"`
 	StartingHeight int32   `json:"startingheight"`
+	CurrentHeight  int32   `json:"currentheight"`
```
Comment (davecgh) [inline jsonresults.go]: I believe this needs to be `omitempty` otherwise it won't work with BC.

Comment (Roasbeef) [inline jsonresults.go]: Correct, added the `omitempty` tag.

**Comment by davecgh** [comment]
OK on code.  Please squash to a single commit.

**Comment by Roasbeef** [comment]
Done.

**Pull request merged**